src/collector/api_server.py

```diff
@@ -102,3 +103,40 @@ def collect(self):
               for m in metrics.values()]
       self._db_con.write_data(data)
       time.sleep(wait_time)
+
+  def add_topic(self):
+    # On the situation where a new solar panel is introduced, this method
+    # adds the corresponding topic names to the database.
+    topic_names = [
```

Comment:
Known metrics are available as a property on the panel_accessor (see panel_accessor.metrics). Better not to hard-code these.